src/cli/onefuzz/api.py

```diff
@@ -1557,6 +1557,20 @@ def list(self) -> responses.ProxyList:
         return self._req_model("GET", responses.ProxyList, data=requests.ProxyGet())
 
 
+class InstanceConfigCmd(Endpoint):
```

Comment:
What does the error message look like when this command fails?

Reply:
The same way all errors look at the CLI.

We generate a 400 error.  The JSON is printed.